eth_crowdfund_be/eth_crowdfund_api/src/models/Campaign.py

```diff
@@ -22,14 +19,11 @@ class Campaign(db.Model):
   expiration = db.Column(db.DateTime)
   created_at = db.Column(db.DateTime)
   updated_at = db.Column(db.DateTime)
-  requests = db.relationship('Request', backref='campaigns')
+  # requests = db.relationship('Request', backref='campaigns')
```

Comment:
We need to revisit this. I received errors that no request was given when trying to create a `campaign`. There shouldn't be any request attached when a Campaign is created.